CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 cmake_minimum_required(VERSION 3.22 FATAL_ERROR)
 
-# CMAKE
```

Comment:
Don`t remove this, this is for people that runs too old systems that doesn't have newer version of cmake

Reply:
Does this comment make any for the build tools?

Reply:
No, it's just to help people when they use an older versions and need to manually update their cmake, so they can check the cmake file and easily find how to upgrade it easier. Also it's a reference to use when people doesn't know what to do and we see the error is because their cmake is older than the one that we need to make the build

Reply:
In this case I would recommend asking the user to upgrade their CMake, and list the version 3.22 as required on the pre requirements, I don't think this comment makes sense here.